README.md

```diff
@@ -142,17 +152,29 @@ If you prefer commercial support, please contact [containo.us](https://containo.
 - We use [Semantic Versioning](http://semver.org/)
 
 
+## Contributing
+
+Please refer to [contributing documentation](.github/CONTRIBUTING.md).
+
+
+### Code of Conduct
+
+Please note that this project is released with a [Contributor Code of Conduct](CODE_OF_CONDUCT.md).
+By participating in this project you agree to abide by its terms.
+
+
 ## Maintainers
 
-- Emile Vauge [@emilevauge](https://github.com/emilevauge)
-- Vincent Demeester [@vdemeester](https://github.com/vdemeester)
-- Ed Robinson [@errm](https://github.com/errm)
-- Daniel Tomcej [@dtomcej](https://github.com/dtomcej)
-- Manuel Zapf [@SantoDE](https://github.com/SantoDE)
-- Timo Reimann [@timoreimann](https://github.com/timoreimann)
-- Ludovic Fernandez [@ldez](https://github.com/ldez)
-- Julien Salleyron [@juliens](https://github.com/juliens)
-- Nicolas Mengin [@nmengin](https://github.com/nmengin)
+[Information about process and maintainers](.github/MAINTAINER.md)
+
+
+## Plumbing
+
+- [Oxy](https://github.com/vulcand/oxy): an awesome proxy library made by Mailgun folks
+- [Gorilla mux](https://github.com/gorilla/mux): famous request router
+- [Negroni](https://github.com/urfave/negroni): web middlewares made simple
```

Comment:
We have an issue on this subject: https://github.com/containous/traefik/issues/1730